video/out/hwdec/hwdec_vaapi.c

```diff
@@ -108,6 +110,8 @@ struct priv_owner {
     bool probing_formats; // temporary during init
 
     struct dmabuf_interop dmabuf_interop;
+    wl_format supported_wl_formats[64];
+    int num_supported_wl_formats;
```

Comment:
Use MP_TARRAY_APPEND rather than hard coding an array size here. (This is how `formats` above is handled)

Reply:
thx, fixed.